recipes/nettle/all/conanfile.py

```diff
@@ -47,84 +49,114 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+        try:
+            del self.settings.compiler.libcxx
+        except Exception:
+            pass
+        try:
+            del self.settings.compiler.cppstd
+        except Exception:
+            pass
+        if self.settings.os == "Windows":
+            self.win_bash = True
```

Comment:
no, it depends on build os, not host os. And I advice to move this logic in build_requirements.